lib/trino-hive-formats/src/main/java/io/trino/hive/formats/ion/IonConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.hive.formats.ion;
+
+import java.util.Map;
+
+public final class IonConstants
+{
+    public static final String ION_ENCODING = "ion.encoding";
+    public static final String TEXT_ENCODING = "text";
+    public static final String BINARY_ENCODING = "binary";
+    public static final String ERROR_ENCODING = "error_encoding";
```

Comment:
Is this only used for tests? If yes, then we should move it into some test util file.

Reply:
Yes, it is only used for the test. I'll move it to the test util file.

Reply:
Yea definitely don't include something only for testing here.